docs/rancher/02-setup-k3s/README.md

```diff
@@ -0,0 +1,142 @@
+# Setting Up k3s for Rancher Integration
+
+## Introduction
+
+Prepare your Ubuntu server for Rancher deployment by installing k3s, a lightweight Kubernetes distribution ideal for simplified cluster management. This setup ensures minimal resource usage while providing a robust environment for Rancher.
+
+After completing this guide, you will have:
+
+- Installed k3s on your Ubuntu server, ready for Rancher.
+- Configured your local environment for seamless k3s cluster management.
+
+## Table of Contents
+
+- [Introduction](#introduction)
+- [Prerequisites](#prerequisites)
+- [Step 1 - Installing and Verifying k3s](#step-1---installing-and-verifying-k3s)
+- [Step 2 - Transferring the kubeconfig File](#step-2---transferring-the-kubeconfig-file)
+- [Step 3 - Customizing the kubeconfig File](#step-3---customizing-the-kubeconfig-file)
+- [Step 4 - Merging Configuration Files for Cluster Management](#step-4---merging-configuration-files-for-cluster-management)
+- [Conclusion](#conclusion)
+
+## Prerequisites
+
+Before starting, ensure you have:
+
+1. An Ubuntu server set up and secured. For instructions, refer to "[How to Set Up and Secure Your Ubuntu Server](../01-setup-ubuntu/README.md)"
+2. Terminal or SSH client access to your local machine.
+
+## Step 1 - Installing and Verifying k3s
+
+Install k3s on Ubuntu to kickstart your Rancher setup:
+
+```bash
+curl -sfL https://get.k3s.io | INSTALL_K3S_VERSION=v1.24.14+k3s1 sh -s - server --cluster-init
```

Comment:
You can refer this link here to explain why we're using this version.
https://github.com/rancher/quickstart/tree/master/rancher/rancher-common